gradle.properties

```diff
@@ -67,7 +67,7 @@ ballerinaCommandVersion=1.3.10
 graphqlVersion=0.3.0-20221109-232100-bdb2701
 
 # OpenAPI Module
-openapiVersion=1.3.0
+openapiVersion=1.4.0-20221212-095900-a875286
```

Comment:
Is this the correct fix going forward? If there are no changes planned for 1.4.0, shouldnt the version be kept 1.3.0 and testcases updared to check for version packed in distribution rather than a fixed version?

Reply:
Yeah, this is the correct fix. this version contains the fix we planned to provide in update 4.